packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -574,7 +594,7 @@ export const findExperiment = async ({
  * @param experiment
  * @return event.id
  */
-export const logExperimentCreated = async (
+const _logExperimentCreated = async (
```

Comment:
Style concern: Isn't the leading underscore generally only used when exporting things that are generally meant to be private, usually just for the sole purpose of tests? Since JavaScript doesn't support visibility modifiers, this convention is used in place of that. I don't see this being exported for test, so I don't think it makes sense to use this naming convention.

Reply:
I'm not strongly tied to it. I only did it to help me reason through this implementation and make it easy to understand what was 'private' and what was not.

My understanding however is that it's to denote simply 'private' methods in general. I don't think that the lack of unit tests for these functions is enough to consider them 'not private'.  Here is one article that supports my idea and doesn't mention anything about exporting for tests (https://www.robinwieruch.de/javascript-naming-conventions/#javascript-naming-conventions-private)

>A private variable/function can occur in a JavaScript file as well. This could mean that the variable/function shouldn't be used outside of this file but only internally to compute further business logic for other functions within the same file..

^ This sums up why I used them here. 

(However I will say we do sorely need more unit tests on the backend. )